codalab_service.py

```diff
@@ -213,6 +219,30 @@ def has_callable_default(self):
     CodalabArg(name='use_ssl', help='Use HTTPS instead of HTTP', type=bool, default=False),
     CodalabArg(name='ssl_cert_file', help='Path to the cert file for SSL'),
     CodalabArg(name='ssl_key_file', help='Path to the key file for SSL'),
+    ### Worker manager
+    CodalabArg(name='worker_manager_type', help='Type of worker manager (e.g., aws, azure, slurm)'),
```

Comment:
Can you add a note in the "help" saying that only aws is supported?

Reply:
Done

Reply:
Probably best to use a `choices` list here, I think that way all available choices get printed in the generated help text.